iati/version.py

```diff
@@ -1,7 +1,15 @@
-"""A module containing components that describe the IATI Standard itself (rather than the parts it is made up of)."""
+"""A module containing components that describe the IATI Standard itself (rather than the parts it is made up of).
```

Comment:
I know this is not changed, but I'm not sure what this sentence really means given this is `version.py`.

Reply:
This probably should have changed since the purpose of the module isn't that!

Reply:
9cff808bab4ed11b533c90609caf62e384a023f1